### PR TITLE
[native] Modify README.md in example_face_detection_tflite

### PR DESCRIPTION
**Pull request by cocony12**
Modify README.md in example_face_detection_tflite for git PR test

Signed-off-by: cocony12 <apap8510@naver.com>

---
# [Template] PR Description

In general, github system automatically copies your commit message for your convenience.
Please remove unused part of the template after writing your own PR description with this template.
```bash
$ git commit -s filename1 filename2 ... [enter]

Summarize changes in around 50 characters or less

More detailed explanatory text, if necessary. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical;
various tools like `log`, `shortlog` and `rebase` can get confused 
if you run the two together.

Further paragraphs come after blank lines.

**Changes proposed in this PR:**
- Bullet points are okay, too
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions vary here.

Resolves: #123
See also: #456, #789

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

**How to evaluate:**
1. Describe how to evaluate in order to be reproduced by reviewer(s).

Add signed-off message automatically by running **$git commit -s ...** command.

$ git push origin <your_branch_name>
```

### CODE REVIEW

**Review by dreamfinder2019 — APPROVED** [review]

File: native/example_face_detection_tflite/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## face_detection
 ### Introduction
 
-This example passes camera video stream to a neural network using **tensor_filter**. The neural network detects faces of people in input stream. A mosaic filter is applied to the detected people's face.
+This example passes camera video stream to a neural network using **tensor_filter**. The neural network detects faces of people in input stream. A mosaic filter is applied to the detected people's face. If the detected face is not one, it is applied to the faces except the largest detected face.
```
Comment (myungjoo) [inline native/example_face_detection_tflite/README.md:4]: Rephrase: ```If there are multiple faces detected, the mechanism is applied to the faces that are not the largest.```

But.. you are applying to the face if there is only one face detected??? Isn't it more natural not to mosaic the face if there is only one?

Comment (cocony12) [reply native/example_face_detection_tflite/README.md:4]: I will consider it with my team members.

**Review by soo-ni — APPROVED** [review]
LGTM

**Review by dreamfinder2019 — APPROVED** [review]

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**